packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -620,6 +620,16 @@ class EditableText extends StatefulWidget {
   /// context, the English phrase will be on the right and the Hebrew phrase on
   /// its left.
   ///
+  /// When text from an LTR language is entered into an RTL field, or RTL text
+  /// is entered into an LTR field,
+  /// [LRM](https://en.wikipedia.org/wiki/Left-to-right_mark) or
+  /// [RLM](https://en.wikipedia.org/wiki/Right-to-left_mark) characters will be
+  /// inserted alongside whitespace characters, respectively. For example, LRM
+  /// characters will be inserted when an English keyboard is used to enter text
+  /// that includes spaces into an app with a Hebrew locale with RTL
```

Comment:
What do you mean by "that includes spaces"? Does this only happen if I hit the space bar on the english keyboard?

Reply:
Yes that's right, for the case that the input is RTL.  I'll clarify.